packages/react/src/button/docs/overview.mdx

```diff
@@ -195,3 +195,17 @@ On mobile, buttons will be stacked on top of one another and span the full width
 	<Button variant="tertiary">Tertiary</Button>
 </ButtonGroup>
 ```
+
+## Focus ring
+
+A button will receive a focus ring when navigated to via the keyboard. If you need to display a focus ring outside of these interactions, e.g. when navigating to a button via a link, use the `focusRingFor` prop.
+
+```jsx live
+<Stack gap={2}>
+	<TextLink href="#focus-button-via-link">Focus the button</TextLink>
```

Comment:
Should we also add an example of focusing with a ref? Would that be the normal use-case?

Reply:
Yeah, we can do that.

- [ ] Example of focusing with a ref